Tests/AWSSDKSwiftCoreTests/QueryEncoderTests.swift

```diff
@@ -128,6 +128,15 @@ class QueryEncoderTests: XCTestCase {
         testQuery(test, query:"A.1.entry=1&A.1.name=first")
     }
     
+    func testDataBlobEncode() {
+        struct Test : AWSShape {
+            static let _members = [AWSShapeMember(label: "A", required: true, type: .blob)]
+            let a : Data
+        }
+        let test = Test(a:Data("Testing".utf8))
+        testQuery(test, query:"A=VGVzdGluZw%253D%253D")
```

Comment:
We want to test that the QueryEncoder returns the correct dictionary. Create the dictionary you would expect and then create a string from it as we for the queries and use that as your comparison
```swift
let result = ["A": Data("Testing".utf8).base64EncodedString()]
testQuery(test, query: queryString(dictionary: result))
```